packages/app/src/settings/SettingsContainer.cy.tsx

```diff
@@ -5,8 +5,12 @@ import SettingsContainer from './SettingsContainer.vue'
 describe('<SettingsContainer />', { viewportHeight: 800, viewportWidth: 900 }, () => {
   const mountSettingsContainer = () => cy.mountFragment(SettingsContainerFragmentDoc, { render: (gql) => <SettingsContainer gql={gql} /> })
 
-  it('renders', () => {
+  it('renders sections collapsed by default', () => {
     mountSettingsContainer()
```

Comment:
nitpick: I know you didn't write this, but is there any reason we couldn't restructure `mountSettingsContainer` into a `beforeEach` so it doesn't have to get manually called in each test?